R/assign.R

```diff
@@ -0,0 +1,473 @@
+#' tracks individual plants through time
+#'
+#' @return
+#' @export
+#'
+#' @examples
+
+# required packages -------------------------------------------------------
+library(sf)
+library(mapview) #don't actually need for function, just for checking
+library(dplyr)
+library(tidyr)
```

Comment:
In the Roxygen documentation use `@import tidyr` so the functions in that package are available for use in your package and so it is known by CRAN and the user the dependencies of your package are.